upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -607,7 +613,164 @@ public function apply_deleteflashsanitizer(): bool
+
+    /**
+     * Check if the session PRIMARY KEY already uses a 200-char prefix on sess_id.
+     *
+     * @return bool true if already normalised (no action needed)
+     */
+    public function check_normalizesessionpk(): bool
+    {
+        $table = $this->db->prefix('session');
+
+        return 200 === $this->indexPrefixLength($table, 'PRIMARY', 'sess_id');
+    }
```

PHPMD (2.15.0)

[error] The method check_normalizesessionpk is not named in camelCase. (undefined)

(CamelCaseMethodName)